src/lib/integration_test_cloud_engine/coda_automation.ml

```diff
@@ -99,6 +100,7 @@ module Network_config = struct
     let project_id = "o1labs-192920" in
     let cluster_id = "gke_o1labs-192920_us-west1_mina-integration-west1" in
     let cluster_name = "mina-integration-west1" in
+    let k8s_context = cluster_id in
```

Comment:
actually looks like `cluster_id` property has the value you want for k8s_context